pyproject.toml

```diff
@@ -73,6 +73,36 @@ sopel-plugins = "sopel.cli.plugins:main"
 [project.entry-points.pytest11]
 pytest-sopel = "sopel.tests.pytest_plugin"
 
+[tool.flake8]
+max-line-length = 79
+no-accept-encodings = true
+type-checking-exempt-modules = "typing"
+exclude = [
+    ".venv",
```

Comment:
This one in particular might be user-specific, but idk how practical it is to layer a local config on top of the project one.